### PR TITLE
1.1

### PR DESCRIPTION
**Pull request by rodinux**
## Problem
- *Upgrade to version 1.1.4*
-  *Add php surry repository  extra dependencies*

## Solution
- *I try use some scripts used before in old nextcloud commits and adapt them*

## PR Status
- [ ] Code finished.
- [x] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ x ] Upgrade from last version tested.
- [ x ] Can be reviewed and tested.

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

**Comment by rodinux** [comment]
update to new versions 1.1 with php7.4... but I have some doubt about the scripts... on a personal server I can't change the domain url...

File: scripts/install [hunk 1]
```diff
@@ -96,7 +106,7 @@ ynh_script_progression --message="Configuring permissions..." --weight=8
 # Make app public if necessary
 if [ $is_public -eq 1 ]
 then
-  ynh_permission_update --permission="main" --add="visitors"
+  ynh_permission_update --permission "main" --add "visitors"
```
Comment (ericgaspar) [inline scripts/install:109]:
```suggestion
  ynh_permission_update --permission="main" --add="visitors"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
You can leave the equal sign.

File: README_fr.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Garradin pour YunoHost
 
-[![Niveau d'intégration](https://dash.yunohost.org/integration/garradin.svg)](https://dash.yunohost.org/appci/app/garradin) ![](https://ci-apps.yunohost.org/ci/badges/garradin.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/garradin.maintain.svg)  
+[![Niveau d'intégration](https://dash.yunohost.org/integration/garradin.svg)](https://dash.yunohost.org/appci/app/garradin) ![](https://ci-apps.yunohost.org/ci/badges/garradin.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/garradin.maintain.svg)
```
[the remark this answers is not in the document]
Comment (rodinux) [reply README_fr.md:3]: Ok

**Comment by rodinux** [comment]
!testme

**Comment by rodinux** [comment]
!testme

**Pull request merged**